appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -189,13 +190,13 @@
   // Remembers the current View
   static final int DESIGNER = 0;
   static final int PROJECTS = 1;
-  private static final int GALLERY = 2;
+  static final int GALLERY = 2;
```

Comment:
I think this was due to some changes that I made in TopToolbar.java. However, I just reviewed all of my changes, and I can't find GALLERY used, so I'll put the private keyword back on.